src/routes/me/index.ts

```diff
@@ -250,3 +243,54 @@ export const mint = (
       status: 201
     };
   });
+
+export const migrate = (db: Firestore, members: CollectionReference) =>
+  createApiRoute<MigrateApiEndpoint>(async (call, loggedInMemberToken) => {
+    const { mobileNumber } = call.body;
+
+    if (!mobileNumber) {
+      throw new ApiError(
+        httpStatus.BAD_REQUEST,
+        "You must supply a mobile number."
+      );
+    }
+
+    let phoneNumberLookup: any;
+    try {
+      phoneNumberLookup = await twilioClient.lookups
+        .phoneNumbers(mobileNumber)
+        .fetch();
+    } catch (e) {
+      throw new ApiError(
+        httpStatus.BAD_REQUEST,
+        "The supplied mobile number could not be validated."
+      );
+    }
+
+    if (!phoneNumberLookup || !phoneNumberLookup.phoneNumber) {
```

Comment:
I think we also want to check that the type of the phone number isn't voip, otherwise this isn't a super useful use of twilio i think because they already wouldn't be able to satisfy the firebase login regardless

Reply:
see #53, also needs to be done for that

Reply:
added mobile only check.